### PR TITLE
translate(pt/game-identification)

### PR DESCRIPTION
**Pull request by G3mha**
Translates the file `docs/pt/developer-docs/game-identification.md` to pt-BR, and adds it to the sidebar.

_PS: Some links to other docs are still in English, since they haven't been translated yet._

### CODE REVIEW

File: docs/pt/developer-docs/game-identification.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+## GCE
+
+- **Vectrex** usa o checksum MD5.
+
+## Magnavox
+
+- **Odyssey2** usa o checksum MD5.
+
+## Mattell
```
Comment (wescopeland) [inline docs/pt/developer-docs/game-identification.md:75]: Should this be with one "l" or two "l"'s? The company name is "Mattel".

Comment (G3mha) [reply docs/pt/developer-docs/game-identification.md:75]: @wescopeland The correct one is MATTEL (https://en.wikipedia.org/wiki/Mattel). I reserved these PRs just for mere translation, I was thinking on doing corrections in separate PRs. But if you authorize me I can correct in both places, any time I stumble upon an error, or inconsistency. What do you prefer?

Comment (wescopeland) [reply docs/pt/developer-docs/game-identification.md:75]: I am much more relaxed in docs than in RAWeb. I prefer whatever you find easiest :-)

Comment (G3mha) [reply docs/pt/developer-docs/game-identification.md:75]: @wescopeland Sounds good! If you don't mind, I prefer to open a PR with those documentation corrections. That way I can earn more points in my course, by accumulating those small changes into a larger contrib.

Comment (wescopeland) [reply docs/pt/developer-docs/game-identification.md:75]: I understand. That should be fine.

**Review by wescopeland — APPROVED** [review]

**Pull request merged**